contrib/scripts/print-downgrade-version.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# A utility script to print the branch name of the previous stable release.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+VERSION="$(cat "$SCRIPT_DIR/../../VERSION")"
+if [[ $VERSION =~ ([0-9^]+)\.([0-9^]+)\..* ]] ; then
+    major=${BASH_REMATCH[1]}
+    minor=${BASH_REMATCH[2]}
+    ((minor--))
```

Comment:
Is it possible that `minor` will be zero?

Reply:
yes it is possible when cilium releases v2.0.

Reply:
> yes it is possible when cilium releases v2.0.

So, then the downgraded version will be: `v2.-1.` right?

Reply:
yeah this thing will fail when we get to cilium v2.0. let's fix the script when that happens 🥰